index.js

```diff
@@ -40,14 +40,14 @@ function screenshot(id, options = {}) {
 
   if (options.bounds) {
     if (typeof options.bounds !== 'object') {
-      throw new TypeError(`'bounds' must be a number`)
-    } else if (!options.bounds.x || typeof options.bounds.x !== 'number') {
```

Comment:
let's replace this check as well as the others with `Object.hasOwnProperty` or `x in options.bounds` instead of deleting it